### ISSUE TITLE
Compiling to static results in error

### ISSUE THREAD

**Issue by nate0001** [reactql/kit#85]
I'm getting this error when trying to load the static build

```
chunk-manifest.json:1 Uncaught SyntaxError: Unexpected token <
vendor.17afe5871716dcce45aa.js:1 Uncaught ReferenceError: webpackJsonp is not defined
    at vendor.17afe5871716dcce45aa.js:1
(anonymous) @ vendor.17afe5871716dcce45aa.js:1
browser.8ab8188d694b8df320d2.js:1 Uncaught ReferenceError: webpackJsonp is not defined
    at browser.8ab8188d694b8df320d2.js:1
(anonymous) @ browser.8ab8188d694b8df320d2.js:1
```

**Comment by leebenson** [comment]
See https://github.com/soundcloud/chunk-manifest-webpack-plugin/issues/47#issuecomment-320508231

It can be fixed per https://github.com/reactql/kit/issues/55#issuecomment-328696200

The above will be pushed with the next release, but you can use that directly in the meantime.

**Comment by leebenson** [comment]
This has been fixed as of [2.9.0](https://github.com/reactql/kit/releases/tag/2.9.0) btw
